src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,45 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface TodoModalProps {
+  handleCloseModal: () => void;
+  selectedUserId: number;
+  selectedTodo: Todo | null;
+}
+export const TodoModal: React.FC<TodoModalProps> = ({
+  handleCloseModal,
+  selectedUserId,
+  selectedTodo,
+}) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  async function handleLoadUserInfo(userId: number) {
+    const loadedUser = await getUser(userId);
```

Comment:
Consider adding error handling for the `getUser` API call to manage scenarios where the user data cannot be fetched. This will improve the robustness of the component.